src/Finder/FinderRegistry.php

```diff
@@ -25,6 +25,9 @@ public static function getAllVariants(): array
             new DummyCpuCoreFinder(1),
             new HwLogicalFinder(),
             new HwPhysicalFinder(),
+            new OnlyOnLinuxFinder(
```

Comment:
If we are not sure I would leave is out; but maybe could be tested on a Windows machine with WSL I have no clue. For sure it's not on OSX though

Reply:
In any case in the `getAllVariants()` I would not add `OnlyOnLinuxFinder` for sure, but I would add `LscpuFinder` with and without physical.

You can inspect the results in the GA "Inspection"

Reply:
What about the defaults? Can also separate into a LscpuLogicalFinder and LscpuPhysicalFinder if you think it aligns better with the Hw* finders.

And sorry, what is GA ? I've only run the scripts in bin and the tests.

Reply:
Sorry I shouldn't be lazy about it, GA=GitHub Actions, e.g. https://github.com/theofidry/cpu-core-counter/actions/runs/3680841567/jobs/6246037753

Reply:
By the way that also gives you a quick way to check the output of your commands on different platforms, which can help for the unit tests

Reply:
Yes it looks pretty good to be honest. There are some errors as I see, which I will address later today (limited time these days).

Reply:
I think coding style and e2e should be fixed now.